frontend/src/component/project/Project/ProjectEnterpriseSettingsForm/ProjectEnterpriseSettingsForm.tsx

```diff
@@ -229,6 +248,7 @@ const ProjectEnterpriseSettingsForm: React.FC<
             onSubmit={submitEvent => {
                 handleSubmit(submitEvent);
                 trackPattern(featureNamingPattern);
+                trackModePattern(projectMode);
```

Comment:
I think these should be in the handleSubmit function - you would only want to track successful submits